OpenShift/helm/nexus-iq/Chart.yaml

```diff
@@ -1,9 +1,25 @@
-apiVersion: v1
-appVersion: 1.84.0
-description: A Helm chart for Kubernetes
-name: iqserver
-version: 0.1.0
+apiVersion: v2
```

Comment:
This declares we need helm 3, right? I wonder if maybe we end up needing to keep helm 2 and helm 3 charts separate for a bit. The operator we built on these may only support helm 2. helm 3 support is supposed to be coming, but I'm not sure it's here yet. I'll have to check back.

Reply:
This looks like a deal-breaker to me unless we receive positive confirmation from RedHat that Helm 3 is now supported.

Reply:
I thought I shared the Operator SDK release notes that show helm 3 support was added.